protob/messages.proto

```diff
@@ -387,6 +387,7 @@ message RecoveryDevice {
 	optional string language = 4 [default='english'];	// device language
 	optional string label = 5;				// device label
 	optional bool enforce_wordlist = 6;			// enforce BIP-39 wordlist during the process
+	optional RecoveryDeviceType type = 7;			// recovery type (scrambled words/matrix)
```

Comment:
Field number 7 conflicts with a field number assigned by KeepKey. Would you consider using a different field number to avoid the conflict? As a maintainer of 2 third party wallet integrations (Multibit and a chrome client), I'm concerned that it will complicate integration if a common protobuf layout can't be used for both devices. 
See https://github.com/keepkey/device-protocol/blob/master/messages.proto#L362